definitions/contested/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -961,7 +961,7 @@
   {
     "LiveFrom": "01/06/2020",
     "CaseTypeID": "FinancialRemedyContested",
-    "CaseEventID": "FR_generalLetter",
```

Comment:
was this a bug or is this a case event that can now be removed?

Reply:
it was originally done adding the judge to this event, so he was granted access to the existing FR_generalLetter event.
However as the judge needs access on different states we needed to duplicate the event for the judge only. So this removes the judges access to the original event and gives him access instead to the judge specific one